google-cloud-pubsub/test/google/cloud/pubsub/subscription_test.rb

```diff
@@ -61,17 +61,6 @@
     mock.verify
   end
 
-  it "can delete itself" do
-    del_res = nil
-    mock = Minitest::Mock.new
-    mock.expect :delete_subscription, del_res, [subscription_path(subscription_name), options: default_options]
-    pubsub.service.mocked_subscriber = mock
-
-    subscription.delete
-
-    mock.verify
-  end
-
```

Comment:
Removed because it is a duplicate of the same test in `subscription/delete_test.rb`.